tests/apps/courses/test_cms_plugins_blogpost.py

```diff
@@ -0,0 +1,176 @@
+# -*- coding: utf-8 -*-
+"""
+Unit tests for the BlogPost plugin and its model
+"""
+from django import forms
+from django.conf import settings
+
+from cms.api import add_plugin, create_page
+from cms.test_utils.testcases import CMSTestCase
+from djangocms_picture.cms_plugins import PicturePlugin
+
+from richie.apps.core.factories import FilerImageFactory, UserFactory
+from richie.apps.core.helpers import create_i18n_page
+from richie.apps.courses.cms_plugins import BlogPostPlugin
+from richie.apps.courses.factories import BlogPostFactory
+from richie.apps.courses.models import BlogPostPluginModel
+
+
+class BlogPostPluginTestCase(CMSTestCase):
+    """
+    Test that BlogPostPlugin correctly displays a BlogPost's page placeholders content
+    """
+
+    def test_cms_plugins_blogpost_form_page_choices(self):
+        """
+        The form to create a blogpost plugin should only list blogpost pages
+        in the select box.
+        """
+
+        class BlogPostPluginModelForm(forms.ModelForm):
+            """A form for testing the choices in the select box"""
+
+            class Meta:
+                model = BlogPostPluginModel
+                fields = ["page"]
+
+        blogpost = BlogPostFactory()
+        other_page_title = "other page"
+        create_page(other_page_title, "richie/fullwidth.html", settings.LANGUAGE_CODE)
+        plugin_form = BlogPostPluginModelForm()
+        self.assertIn(blogpost.extended_object.get_title(), plugin_form.as_table())
```

Comment:
As explained in this PR https://github.com/openfun/richie/pull/509/files#diff-67a30e02803ca05e2185d5da408b5847R89 you should count to make sure there are no duplicates:
```suggestion
        rendered_form = plugin_form.as_table()
        self.assertEqual(rendered_form.count(blogpost.extended_object.get_title()), 1)
```